components/ProductMatrix.vue

```diff
@@ -45,6 +45,7 @@ export default Vue.extend({
     loading: 'products/isLoading'
   }),
   mounted() {
+    this.$chat('c', 'Hej! 👋')
```

Comment:
Let's make this a general event instead. What do you think?
`this.$event("mounted",{ component: "ProductMatrix", foo: "bar" }`

Then, when this get triggered we can attach chat, console.log.debug, Google Analytics reporting etc.